adoptopenjdk-api-v3-frontend/pom.xml

```diff
@@ -76,9 +76,12 @@
         <dependency>
             <groupId>net.adoptopenjdk.api</groupId>
             <artifactId>adoptopenjdk-api-v3-updater</artifactId>
-            <version>${project.version}</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I'd argue this is not utils but in fact a monitoring or telemetry module no?

Reply:
that's a good shout, utils is generally a smell and could easily become a large bucket of stuff - creating a module focused on the purpose of this feature feels like a good move